app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/patient_search_and_verify_workflow.cy.js

```diff
@@ -63,6 +65,7 @@ describe('GP Workflow: Patient search and verify', () => {
                     'Check these patient details match the records or attachments you plan to use',
                 );
                 cy.get('#verify-submit').click();
+                cy.title().should('eq', 'Available records - Digital Lloyd George records');
```

Comment:
This test seems to be failing. Looks like when the patient is inactive, the logic at PatientResultPage.tsx line 32:36 would redirect us to the old ARF upload page instead of LG view page.